spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/StreamParser.java

```diff
@@ -145,8 +146,7 @@ private String eatStreamName() {
 	private StreamNode eatStream() {
 		String streamName = eatStreamName();
 		SourceChannelNode sourceChannelNode = eatSourceChannel();
-
-		// This construct: queue:foo > topic:bar is a source then a sink channel
+		// This construct: foo > bar is a source then a sink channel
```

Comment:
This should still have `:foo` and `:bar`, right?

Also, we should avoid referring to those as "channels" from now on, i.e. we need to realign terminology across the board so that "channel" is reserved for the actual `MessageChannel` component *within an app* that is being bound to the broker via a binding. Whenever we discuss the broker side, we should refer to it as simply "destination" - and whereas we used to use "named channel" as a way to distinguish from pipes, I believe we should avoid "named" as an adjective from now on as well, since it's redundant. Every destination is "named". The destinations that are established for pipe bindings just happen to use a naming convention of `<stream>.<label>`now.